subworkflows/local/amp_prediction.nf

```diff
@@ -0,0 +1,27 @@
+//
+// Check input samplesheet and get read channels
+//
+
+include { MACREL_CONTIGS } from '../../modules/nf-core/macrel/contigs/main'
+include { AMPIR_PEPTIDES } from '../../modules/foo'
+include { MERGE_PREDICTIONS } from '../../modules/local/merge_predictions'
+
+workflow AMP_PREDICTION {
+    take:
+    proteins  // tuple val(meta), path('*.faa')
+
+    main:
+    MACREL_PEPTIDES(proteins)
+    AMPIR_PEPTIDES(proteins)
+    // AMPLIFY
+
+    ch_predictions = MACREL_CONTIGS.out.amp_prediction.join(
```

Comment:
Will this work? How will you know which tool goes with with merge_predictions `--<tool>` flag?